packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -296,16 +294,10 @@ export class InMemoryAnchorService implements AnchorService, AnchorValidator {
     const feed$ = this.#feed.pipe(
       filter((asr) => asr.streamId.equals(streamId) && asr.cid.equals(tip))
     )
-
     if (anchorResponse) {
       return concat(of<AnchorServiceResponse>(anchorResponse), feed$)
     } else {
-      return of<AnchorServiceResponse>({
```

Comment:
wow, that previous behavior definitely seems wrong.  Kind of wild it never was an issue before now.  Good catch